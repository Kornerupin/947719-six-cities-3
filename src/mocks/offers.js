export default [
  {
    image: [
      `img/apartment-01.jpg`,
      `img/apartment-02.jpg`,
      `img/apartment-03.jpg`,
    ],
    type: `Apartment`,
    price: `80`,
    rating: 3.1,
    name: `Beautiful &amp; luxurious apartment at great location`,
    coords: [52.3909553943508, 4.85309666406198]
  },
  {
    image: [`img/room.jpg`],
    type: `Private room`,
    price: `132`,
    rating: 1.8,
    name: `Wood and stone place`,
    coords: [52.369553943508, 4.85309666406198]
  },
  {
    image: [`img/apartment-02.jpg`],
    type: `Apartment`,
    price: `180`,
    rating: 3.8,
    name: `Canal View Prinsengracht`,
    coords: [52.3909553943508, 4.929309666406198]
  },
  {
    image: [`img/apartment-03.jpg`],
    type: `Apartment`,
    price: `80`,
    rating: 4.5,
    name: `Nice, cozy, warm big bed apartment`,
    coords: [52.3809553943508, 4.939309666406198]
  },
];
