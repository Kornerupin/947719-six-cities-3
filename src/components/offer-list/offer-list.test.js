import OfferList from './offer-list';
import offers from '../../mocks/offers';

it(`OfferList /> Should render list of cards`, () => {
  const tree = renderer.create(<OfferList offers={offers} />).toJSON();

  expect(tree).toMatchSnapshot();
});
