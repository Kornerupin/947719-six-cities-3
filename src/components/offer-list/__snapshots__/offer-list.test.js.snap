// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OfferList /> Should render list of offers 1`] = `
Array [
  <article
    className="cities__place-card place-card"
    onMouseOver={[Function]}
  >
    <div
      className="cities__image-wrapper place-card__image-wrapper"
    >
      <a
        href="#"
      >
        <img
          alt="Place image"
          className="place-card__image"
          height="200"
          src="img/apartment-01.jpg"
          width="260"
        />
      </a>
    </div>
    <div
      className="place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            80
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button place-card__bookmark-button--active button"
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height="19"
            id="icon-bookmark"
            viewBox="0 0 17 18"
            width="18"
          >
            <path
              d="M3.993 2.185l.017-.092V2c0-.554.449-1 .99-1h10c.522 0 .957.41.997.923l-2.736 14.59-4.814-2.407-.39-.195-.408.153L1.31 16.44 3.993 2.185z"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            In bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "60%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
        onClick={[Function]}
      >
        <a
          href="#"
        >
          Beautiful &amp; luxurious apartment at great location
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        Apartment
      </p>
    </div>
  </article>,
  <article
    className="cities__place-card place-card"
    onMouseOver={[Function]}
  >
    <div
      className="cities__image-wrapper place-card__image-wrapper"
    >
      <a
        href="#"
      >
        <img
          alt="Place image"
          className="place-card__image"
          height="200"
          src="img/room.jpg"
          width="260"
        />
      </a>
    </div>
    <div
      className="place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            132
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button place-card__bookmark-button--active button"
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height="19"
            id="icon-bookmark"
            viewBox="0 0 17 18"
            width="18"
          >
            <path
              d="M3.993 2.185l.017-.092V2c0-.554.449-1 .99-1h10c.522 0 .957.41.997.923l-2.736 14.59-4.814-2.407-.39-.195-.408.153L1.31 16.44 3.993 2.185z"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            In bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "40%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
        onClick={[Function]}
      >
        <a
          href="#"
        >
          Wood and stone place
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        Private room
      </p>
    </div>
  </article>,
]
`;
